src/api/common/TutanotaConstants.js

```diff
@@ -338,17 +338,6 @@ export const InputFieldType = Object.freeze({
 })
 export type InputFieldTypeEnum = $Values<typeof InputFieldType>;
 
-export const EntropySrc = Object.freeze({
```

Comment:
npm testclient now fails, because EntropySrc is not defined anymore at runtime. Maybe we should not remove the enum object?

Reply:
looks like it was just a refactoring error. should just be `"key"` in `test/client/Suite.js`. The main issue I found was that the enum would be required on the main thread, which would mean we would have to bundle at least part of `tutanota-crypto` into `common` or something. I thought it would be better if we just say everything goes into `worker`